src/app/components/Filters/FieldsetDate.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
```

Comment:
@ktp242 , if you want, you can branch off this PR and use this component I started to work on.

Reply:
But it's okay if you start your own, make it work, and then we can put it all together.